src/Catel.MVVM/Catel.MVVM.Shared/Collections/FastObservableCollection.cs

```diff
@@ -326,6 +422,112 @@ protected override void OnPropertyChanged(PropertyChangedEventArgs e)
                 }
             }
         }
+
+        #region Overrides of ObservableCollection
+        /// <summary>
+        /// Removes all items from the collection.
+        /// </summary>
+        protected override void ClearItems()
+        {
+            // Check
+            if (_suspendChangeNotifications && _suspensionContext.Mode != SuspensionMode.Mixed)
```

Comment:
_suspendChangeNotifications is like a synonym but I can/will change it to be consistent and safe.